recipes/pagmo2/config.yml

```diff
@@ -1,3 +1,5 @@
 versions:
   "2.17.0":
     folder: all
+  "2.8":
+    folder: 2.8
```

Comment:
this folder could have another name, 2.8 is too specific since the same conanfile could be used for all versions before 2.11

Reply:
Let me check, I think it's very possible

Reply:
from the first tests, 2.10 seems to work, I will add package 2.10 instead then. 
Would the folder name "prior_2.11" be ok?